.moon/tasks/typescript.yml

```diff
@@ -1,7 +1,9 @@
 $schema: https://moonrepo.dev/schemas/tasks.json
 tasks:
   lint:
-    command: eslint .
+    command: eslint . --format unix --quiet --pass-on-no-patterns
     inputs:
-      - .eslintrc.cjs
+      - /eslint.config.mjs
+      # Is this fine:
```

Comment:
what's your concern?

Reply:
@dmaretskyi might be wise limit it to .js files? but fwiw I'm pretty sure that project isn't even built, it is only linted